src/relay/package.json

```diff
@@ -28,6 +28,7 @@
     "is-ci": "^2.0.0",
     "react-relay": "^7.1.0",
     "relay-compiler": "^7.1.0",
+    "relay-compiler-language-typescript": "^10.1.1",
```

Comment:
does it have to be dependency? `import(optionFromCli)` is no?

Reply:
It doesn't have to be dependency but you'd have to require it manually when you need it. I added it here to simplify the process since it doesn't increase the bundle size or anything (it's a development thingy only for the Relay Compiler). We can tune it later when there is a real concern not to do it.